recipes/objectbox-generator/all/conanfile.py

```diff
@@ -1,48 +1,60 @@
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
 import os
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import copy, download, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.47.0"
+
 
 class PackageConan(ConanFile):
     name = "objectbox-generator"
     description = "ObjectBox Generator based on FlatBuffers schema files (fbs) for C and C++"
     license = "GPL-3.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/objectbox/objectbox-generator"
-    topics = ("database", "code-generator", "objectbox")
+    topics = ("database", "code-generator", "objectbox", "pre-built")
+
+    package_type = "application"
     settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
```

Comment:
```suggestion
```
no_copy_source can be removed as binaries should be downloaded to the build folder directly, so there is nothing to copy from source to build folder.